### ISSUE TITLE
release patch for rtlcss 2.6.x with updated dependency postcss

### ISSUE THREAD

**Issue by evansrobert** [MohammadYounes/rtlcss#237]
Hi, @MohammadYounes, I stumbled upon a vulnerability introduced by package **postcss@6.0.23**:
### Issue Description
When I build my project, I notice that a vulnerability [**CVE-2021-23382**](https://snyk.io/vuln/SNYK-JS-POSTCSS-1255640) detected in package **postcss(<7.0.36,>=8.0.0 <8.2.13)** is directly referenced by **_rtlcss@2.6.2_**.
   However,  **rtlcss@2.6.2** is so popular that a large number of latest versions of active and popular downstream projects depend on it ([41,811 downloads per week](https://www.npmjs.com/package/rtlcss/v/2.6.2?activeTab=versions) and about **34 downstream projects**, e.g., gulp-rtlcss 1.4.2, rtlcss-webpack-plugin 4.0.6, @oroinc/oro-webpack-config-builder 5.1.0-alpha20, @automattic/social-previews 1.1.1, select2-bootstrap-5-theme 1.1.1, etc.). 
In this case, the vulnerability [**CVE-2021-23382**](https://snyk.io/vuln/SNYK-JS-POSTCSS-1255640) can be propagated into these downstream projects and expose security threats to them.
   As you can see, **_rtlcss@2.6.2_** is introduced into the above projects via the following package dependency paths:
(1)`rules_prerender@0.0.9 ➔ @bazel/postcss@0.5.0 ➔ rtlcss@2.6.2 ➔ postcss@6.0.23`
    **......**

I know that it's kind of you to have removed the vulnerability since **_rtlcss@3.0.0_**. But, in fact, the above large amount of downstream projects cannot easily upgrade **rtlcss** from version **_2.6.2_** to _**(>=3.0.0)**_:
    The projects such as **@bazel/postcss**, which introduced **rtlcss@2.6.2**, are not maintained anymore. These unmaintained packages can neither upgrade **rtlcss** nor be easily migrated by the large amount of affected downstream projects.  

   **Given the large number of downstream users, is it possible to release a new patched version with the updated dependency to remove the vulnerability from package rtlcss@2.6.2?**

### Suggested Solution

Since these inactive projects set a version constaint **2.6.\*** for **rtlcss** on the above vulnerable dependency paths, if **rtlcss** removes the vulnerability from **2.6.2** and releases a new patched version **_rtlcss@2.6.3_**, such a vulnerability patch can be automatically propagated into the downstream projects.

In **_rtlcss@2.6.3_**, maybe you can try to perform the following upgrade:
`postcss ^6.0.23 ➔ ^7.0.36`;  
**Note:**
**_postcss@7.0.36(>=7.0.36 <8.0.0)_** has fixed the vulnerability [**CVE-2021-23382**](https://snyk.io/vuln/SNYK-JS-POSTCSS-1255640).

Thank you for your attention to this issue and welcome to share other ways to resolve the issue.

**Comment by MohammadYounes** [comment]
@evansrobert Wouldn't upgrading `postcss ^6.0.23 ➔ ^7.0.36` introduce any breaking changes ?

**Comment by XhmikosR** [comment]
@MohammadYounes yup, plus this has been already fixed by postcs in 7.x, so you can close this issue.
